test/specs/elasticsearch/elasticsearch.ts

```diff
@@ -37,9 +37,9 @@ describe( 'ElasticSearch', function () {
 		let searchResult: SearchResult[];
 
 		const testLabel = 'Testitem';
-		expect( itemLabel.includes( testLabel ) ).toBe( false );
-		expect( itemLabel.toLowerCase().includes( testLabel.toLowerCase() ) ).toBe(
-			true
+		expect( itemLabel ).toEqual( expect.not.stringContaining( testLabel ) );
+		expect( itemLabel.toLowerCase() ).toEqual(
+			expect.stringContaining( testLabel.toLowerCase() )
```

Comment:
I think those nested `expect`s are not really intuitive. Is there a more clear way to express that?

Reply:
The other way to do it would be `expect( actual.includes( expected ) ).toBe( true )` - would we prefer that?

Reply:
...actually, yeah, I think that is better